modules/ztp-du-host-bios-requirements.adoc

```diff
@@ -0,0 +1,75 @@
+// CNF-1500 ZTP
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="configuring_bios_for_du_hosts_{context}"]
+= Configuring BIOS for distributed unit bare-metal hosts
+
+Distributed unit (DU) hosts require certain BIOS settings to be configured before the host can be provisioned.
+
+.Procedure
+. Set the *UEFI/BIOS Boot Mode* to `UEFI` and ensure that the same setting is applied in the `BareMetalHost` CRD.
+
+. Set *Hard drive first* in the host boot sequence order.
++
+[NOTE]
+====
+If `rootDeviceHints` is specified in the `BareMetalHost` CRD, the hard disk must match the hard disk that you configure in the host boot sequence order.
+====
+
+. Apply the BIOS configurations for your hardware. The following table describes a representative BIOS configuration for an Intel® Xeon Skylake or Intel Cascade Lake server, based on the Intel FlexRAN 4G and 5G baseband PHY reference design.
```

Comment:
> Intel®

I don't see other uses of the ® symbol with "Intel" in `master`. Will you confirm that it's required and update the whole PR, if required?

Reply:
https://www.intel.com/content/www/us/en/trademarks/intel.html 

If we follow Intel's guidelines, we would only update the first instance with the registered trademark. But I take your point, I do not see it elsewhere in the repo. Will remove and maybe open a discussion on this.